README.md

```diff
@@ -1,84 +1,46 @@
-# 🔗 LittleLink-Server
+<div align="center">
 
-This project is based on the great work from [littlelink](https://github.com/sethcottle/littlelink)
-It takes the same simple approach to a link page and hosts it within a NodeJS server with React Server Side Rendering, containerized for you to use. Now, customizing `LittleLink` with `littlelink-server` is as easy as passing in some environment variables.  If you need help configuring this, please see [this video that explains everything](https://www.youtube.com/watch?v=42SqfI_AjXU) and a live example at [technotim.live](https://technotim.live/).
-
-## 🚀 Getting Started
-
-## 📍 Supported Links & Buttons & Config
-
-Check the [config](https://github.com/techno-tim/littlelink-server/blob/master/src/config.js) file for all supported buttons and configuration!
-
-## 📈 Analytics Support
-
-### Google Analytics
-
-See [Getting Started with Analytics](https://support.google.com/analytics/answer/1008015?hl=en).  After getting your GA Tracking Id, use your tracking Id as environment variable like `GA_TRACKING_ID=G-XXXXXXXXXX`  (See the example below)
-
-All buttons clicked will be tracked automatically if `GA_TRACKING_ID` exists.
-
-Sample event for YouTube button.
-
-```javascript
-  window.gtag('event', 'youtube-button');
-```
-
-### Umami
-
-See [Adding a website & Collecting data](https://umami.is/docs/collect-data) page to add and generate your tracking code.
-
-Generated tracking code should look like:
-
-```javascript
-<script async defer data-website-id="00000000-1111-2222-3333-444444444444" src="https://your-umami-app.com/umami.js"></script>
-```
+![LittleLink](https://cdn.cottle.cloud/littlelink/social-circle.png)
 
-Use `data-website-id` as environment variable `UMAMI_WEBSITE_ID` and `src` as `UMAMI_APP_URL`.
+## LittleLink-Server
 
-Sample event for YouTube button.
+<h4 align="center">
 
-```javascript
-  window.umami('youtube-button');
-```
+LittleLink is a lightweight DIY alternative to services like [Linktree](https://linktr.ee)
+and [many.link](https://www.google.com).
 
-### Matomo 
+A fork of [littlelink](https://github.com/sethcottle/littlelink).
```

Comment:
Technically, this is no longer a fork.  Maybe "heavily inspired by?  It started as a fork but then I rewrote the whole thing in react.